bundles/search.rst

```diff
@@ -99,10 +99,49 @@ index, and can modify it, before it is compared with the first argument from
 all ``setUrl`` calls. This is especially useful if the `ExpressionLanguage`_
 is used for the generation of the index name.
 
+Reindexing
+----------
+
+Re-indexing is the process of reading all of the documents in the system and
+regenerating their search records. This is necessary when changes are made to
+the metadata and it is desirable to propagate these changes over all of the
+indexed documents / entities in the system -- or when you import new data
+(e.g. from a backup) and need to index that data.
+
+To reindex all entities (Contacts, Media, etc) and documents (Pages, Snippets)
+simply run the following:
+
+.. code-block:: bash
+
+    $ ./app/console massive:search:index:rebuild --env=prod
```

Comment:
Yeah, was written before that.

On Wed, Feb 17, 2016 at 12:24:57AM -0800, Daniel Rotter wrote:
>    In [1]bundles/search.rst:
> 
>  > @@ -99,10 +99,49 @@ index, and can modify it, before it is compared with the first argument from
>  >  all ``setUrl`` calls. This is especially useful if the `ExpressionLanguage`_
>  >  is used for the generation of the index name.
>  >
>  > +Reindexing
>  > +----------
>  > +
>  > +Re-indexing is the process of reading all of the documents in the system and
>  > +regenerating their search records. This is necessary when changes are made to
>  > +the metadata and it is desirable to propagate these changes over all of the
>  > +indexed documents / entities in the system -- or when you import new data
>  > +(e.g. from a backup) and need to index that data.
>  > +
>  > +To reindex all entities (Contacts, Media, etc) and documents (Pages, Snippets)
>  > +simply run the following:
>  > +
>  > +.. code-block:: bash
>  > +
>  > +    $ ./app/console massive:search:index:rebuild --env=prod
> 
>    Don't you want to use the new reindex command here already?
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu-docs/pull/163#discussion_r53132654
>    2. https://github.com/sulu-io/sulu-docs/pull/163/files#r53132654
>    3. https://github.com/sulu-io/sulu-docs/pull/163/files#r53132654